contracts/Automate.sol

```diff
@@ -166,6 +167,43 @@ contract Automate is Gelatofied, Proxied, AutomateStorage, IAutomate {
         );
     }
 
+    function exec1BalanceSimple(
+        address _taskCreator,
+        address _execAddress,
+        bytes memory _execData,
+        bytes32 _taskId,
+        Gelato1BalanceParam calldata _oneBalanceParam,
```

Comment:
I think as discussed with Denis on the Confluence specs, this can now be reduced to correlationId only